test/StringField_test.js

```diff
@@ -773,7 +773,8 @@ describe("StringField", () => {
       );
 
       expect(lengths).eql([
-        121 + 1, // from 1900 to 2020 + undefined
+        // from 1900 to current year + 2 (inclusive) + 1 undefined option
```

Comment:
A nitpick - but would it make sense to extract a function that provides this, to avoid duplication between tests ?

Reply:
That is imo not really necessary, in tests duplication is not so bad.